.rubocop.yml

```diff
@@ -1,9 +1,10 @@
 require:
+  - standard
+plugins:
   - rubocop-capybara
   - rubocop-minitest
   - rubocop-performance
   - rubocop-rails
-  - standard
 
```

Comment:
Silence a warning while running rubocop. rubocop-capybara, rubocop-minitest, rubocop-performance, and rubocop-rails extension supports the new plugin system in rubocop 1.72.